src/main/groovy/com/okta/test/mock/tests/CodeFlowRemoteValidationIT.groovy

```diff
@@ -118,4 +119,7 @@ class CodeFlowRemoteValidationIT extends ApplicationTestRunner {
 
     }
 
+    protected Matcher<?> loginPageMatcher() {
+        return Matchers.equalTo("<html>fake_login_page<html/>")
+    }
 }
```

Comment:
nit: new line

Reply:
New line Isn't needed here (though some people love them, some people hate them :) ), it just depends on the reviewer.  Lets leave it for now, but we should decide one way or the other for the future